caffe2/python/onnx/backend.py

```diff
@@ -874,8 +860,8 @@ def _onnx_model_to_caffe2_net(cls, onnx_model, device, opset_version, include_in
         device_option = get_device_option(Device(device))
 
         onnx_model = onnx.utils.polish_model(onnx_model)
```

Comment:
`polish_model` also invokes `onnx.optimizer.optimize`. Maybe this line should also be removed.
`onnx_model = onnx.utils.polish_model(onnx_model)`

Reply:
Good point. Even though I will also remove `onnx.optimize` in `onnx.utils.polish_model`, I think the purpose of `polish_model` is for `onnx.optimize`. Therefore, it should be remove here. Thanks!